src/azure-cli/azure/cli/command_modules/cosmosdb/_validators.py

```diff
@@ -17,8 +17,13 @@ def validate_failover_policies(ns):
 
 
 def validate_ip_range_filter(ns):
-    if ns.ip_range_filter:
-        ns.ip_range_filter = ",".join(ns.ip_range_filter)
+    """ Extracts multiple comma-separated ip rules """
+    from azure.mgmt.cosmosdb.models import IpAddressOrRange
+    if ns.ip_range_filter is not None:
+        ip_rules_list = []
+        for item in ns.ip_range_filter.split(","):
```

Comment:
so you are expecting that cx use `,` to separate ip rules, but in previous design it is space.

Reply:
In previous design, it was a list of strings, each string is a comma separated values of ip-range filter. I meant to keep it the same way. I would add a loop here as well to iterate on each element on the inputted array. Not intending to make any breaking changes here.

Reply:
I mean for previous design, `--ip-range-filter a b` will work, right? but currently it seems that it doesn't work.